api/Screen.json

```diff
@@ -50,59 +51,1084 @@
+            },
+            "ie": {
+              "version_added": null
+            },
+            "opera": {
+              "version_added": true
+            },
+            "opera_android": {
+              "version_added": true
+            },
+            "safari": {
+              "version_added": true
+            },
+            "safari_ios": {
+              "version_added": null
+            },
+            "samsunginternet_android": {
               "version_added": null
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "left": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Screen/left",
+          "support": {
+            "webview_android": {
+              "version_added": true
+            },
+            "chrome": {
+              "version_added": true
+            },
+            "chrome_android": {
+              "version_added": true
+            },
+            "edge": {
+              "version_added": true,
+              "notes": "Only <code>height</code> and <code>width</code> reflect the current screen; <code>availHeight</code> and <code>availWidth</code> always reflect the main screen; <code>availLeft</code> and <code>availTop</code> are not implemented."
             },
             "edge_mobile": {
               "version_added": null
             },
-            "firefox": [
-              {
-                "version_added": true
-              },
-              {
-                "version_added": true,
-                "prefix": "moz"
-              }
-            ],
-            "firefox_android": [
-              {
-                "version_added": true
-              },
-              {
-                "version_added": true,
-                "prefix": "moz"
-              }
-            ],
+            "firefox": {
+              "version_added": true
+            },
+            "firefox_android": {
+              "version_added": null
+            },
             "ie": {
-              "version_added": "11",
-              "prefix": "ms",
-              "notes": "Not supported on Windows 7."
+              "version_added": null
             },
             "opera": {
-              "version_added": "25"
+              "version_added": true
             },
             "opera_android": {
-              "version_added": false
+              "version_added": true
             },
             "safari": {
-              "version_added": false
+              "version_added": true
             },
             "safari_ios": {
-              "version_added": false
+              "version_added": null
+            },
+            "samsunginternet_android": {
+              "version_added": null
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
```

Comment:
The MDN article says left is non-standard https://developer.mozilla.org/en-US/docs/Web/API/Screen/left